### ISSUE TITLE
Fix go lint issues policy_jiva_k8s.go

### ISSUE THREAD

**Issue by satyamz** [openebs/openebs#2103]
Following file has a lint issue:
[maya/volume/policies/v1/policy_jiva_k8s.go](https://github.com/openebs/maya/blob/master/volume/policies/v1/policy_jiva_k8s.go)

The lint  issue is : 
```
        Line 17: warning: package comment should be of the form "Package v1 ..." (golint)
        Line 58: warning: exported function NewJivaK8sPolicies should have comment or be unexported (golint)


```

Help Links : 
[Contributors guide](https://github.com/openebs/openebs/blob/master/CONTRIBUTING.md)

**Comment by sonapraneeth-a** [comment]
Can I take up this issue? Also, it would be great if you could tell what this "NewJivaK8sPolicies" does
so that I can add the appropriate comment

**Comment by sonapraneeth-a** [comment]
Fixed the issue in PR [#740](https://github.com/openebs/maya/pull/740)
